### PR TITLE
Salto 1285 expose progress information in the cli

### PR DESCRIPTION
**Pull request by omrifarkash**
_Release Notes_: 
Added fetching progress output in the CLI

### CODE REVIEW

File: packages/cli/src/commands/fetch.ts [hunk 1]
```diff
@@ -73,6 +73,9 @@ export const fetchCommand = async (
     bindedOutputline(formatFetchHeader())
   })
 
+  fetchProgress.on('adapterProgress', (adapterName: string, _operationName: AdapterOperationName, progress: Progress) =>
+    outputLine(subHeader(`       - ${adapterName} adapter: ${progress.message}`), output))
```
Comment (adamro) [inline packages/cli/src/commands/fetch.ts:77]: This kind of text building and formatting should  done in the `formatter` (using prompts and subHeader inside it).
The code here should be closer to `bindedOutputline(formatAdapterProgress(adapterName, progress.message))` (with `formatAdapterProgress` being a function in the `formatter` file)

Look at the `formatter` file for examples. 
(I am not even sure why `subHeader` is exported from formatter but it should logically be used only inside it.
The spaces and the rest of the text structure should be using the Prompts file.)

Comment (omrifarkash) [reply packages/cli/src/commands/fetch.ts]: changed it and the above comment as well

File: packages/cli/src/prompts.ts [hunk 2]
```diff
@@ -329,4 +329,7 @@ ${Prompts.LIST_IDS(ids)}
 
   public static readonly GO_TO_SERVICE_NOT_SUPPORTED_FOR_ELEMENT =
     (elementId: string): string => `Go to service is not supported for element ${elementId}`
+
+  public static readonly FETCH_PROGRESSING_MESSAGES =
+    (adapterName: string, progressMessage: string): string => `       - ${adapterName} adapter: ${progressMessage}`
```
Comment (adamro) [inline packages/cli/src/prompts.ts:334]: look at the `indent` function for the spaces

File: packages/cli/src/commands/fetch.ts [hunk 3]
```diff
@@ -73,6 +73,9 @@ export const fetchCommand = async (
     bindedOutputline(formatFetchHeader())
   })
 
+  fetchProgress.on('adapterProgress', (adapterName: string, _operationName: AdapterOperationName, progress: Progress) =>
```
[the remark this answers is not in the document]
Comment (adamro) [reply packages/cli/src/commands/fetch.ts:76]: Again, pure style, I prefer this - 

```
fetchProgress.on(
    'adapterProgress',
    (adapterName: string, _operationName: AdapterOperationName, progress: Progress) =>
        (bindedOutputline(formatAdapterProgress(adapterName, progress.message))),
)
```

(It looks like the rest of the file is actually more similar to what you did so U can completely keep it if you think it's better)

**Review by adamro — APPROVED** [review]

**Pull request merged**